packages/cli/src/lib/project/dotFolder.ts

```diff
@@ -20,8 +20,8 @@ export class DotFolder {
 }
 
 export class DotFolderConfig {
-  public constructor(public ownerFolder: DotFolder) {
-    this.ensureFileExists();
+  public constructor(public ownerFolder: DotFolder, orgId?: string) {
```

Comment:
In what scenarios would that `orgId` be undefined/unavailable ?

Reply:
> In what scenarios would that `orgId` be undefined/unavailable ?

For example, I spotted this problem when doing the following path:
1. `ui:resources:model:create`
2. Saved a SPM
3. Try to consume it with a `ui:resources:pull`, the config.json contained an empty orgId.

To be more broad, it will happen if you run any `ui:resources` commands in an empty folder (more precisely, a folder without a `.coveo` dir in it).